sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamBigQuerySqlDialect.java

```diff
@@ -222,6 +222,14 @@ public void unparseCall(
         }
         writer.endFunCall(frame);
         break;
+      case ROW:
```

Comment:
nit: ideally such `unparse` should happen within `operator` classes. But since we implement it in Beam it is just ok.

Reply:
I thought about replacing [`SqlRowOperator`](https://calcite.apache.org/apidocs/org/apache/calcite/sql/fun/SqlRowOperator.html) with our own custom operator. But I like this better, because it only overrides unparsing without affecting the rest of the operator code.